storage/storage_image.go

```diff
@@ -360,6 +361,22 @@ func (s *storageImageDestination) HasThreadSafePutBlob() bool {
 	return true
 }
 
+// SupportsBlobLocks indicates whether the ImageDestination supports blob
+// locking.
+func (d *storageImageDestination) SupportsBlobLocks() bool {
+	return false
+}
+
+// NOOP for this type.
+func (s *storageImageDestination) LockBlob(b types.BlobInfo) error {
+	return s.imageRef.transport.store.LockDigest(b.Digest)
+}
+
+// NOOP for this type.
```

Comment:
(This is not a documentation that would help the caller; if it should exist, it probably should be inside the function body, so that `go doc` does not generate it.)

… besides being incorrect in this case. If you are going to copy&paste something everywhere, copy&paste the interface documentation.

Reply:
Done.